tests/stave_tests.ts

```diff
@@ -640,7 +641,7 @@ function drawVoltaModifier(options: TestOptions, contextBuilder: ContextBuilder)
   // mm5.addModifier(new Repetition(Repetition.type.DS, mm4.getX() + mm4.getWidth(), 50), StaveModifierPosition.RIGHT);
   mm6.setRepetitionType(Repetition.type.DS);
   mm6.setMeasure(6);
-  mm6.setSection('E', 0);
+  mm6.addModifier(new StaveSection('E'));
```

Comment:
I noticed all the calls to .setSection() have been replaced with addModifier(new StaveSection(...)).

Does the older API still work? setSection has been around since VexFlow 3.0.9 so hopefully the older approach still works (with good defaults).

Do we have a wiki document to document such changes? That way, people coming from VexFlow 4.x or 3.x can know that .addModifier(new StaveSection(...)) is the newer preferred approach. (Although I think the previous API seems simpler, and should be kept around).

Reply:
Here is the API in VexFlow 3.0.9:

https://github.com/0xfe/vexflow/blob/3.0.9/tests/stave_tests.js

```js
var staveBar1 = new VF.Stave(10, 50, 200);
staveBar1.setBegBarType(VF.Barline.type.REPEAT_BEGIN);
staveBar1.setEndBarType(VF.Barline.type.DOUBLE);
staveBar1.setSection('A', 0);
staveBar1.addClef('treble').setContext(ctx).draw();
```

Reply:
Calling setSection does not provide any pointer the StaveSection and this why I decided to use the standard addModifier. setSection is still available but does not allow to change the padding.

Reply:
setSection now also being tested.